Sources/SwiftSyntaxBuilderGeneration/Node.swift

```diff
@@ -84,9 +85,30 @@ class Node {
     self.description = description
 
     self.traits = traits
-    self.children = children
     self.baseKind = kind
 
+    if kind == "SyntaxCollection" {
+      self.children = children
+    } else {
+      // Add implicitly generated GarbageNodes children between
+      // any two defined children
+      self.children = children.enumerated().flatMap { (i, child) in
+        [
+          Child(
+            name: i == 0
+              ? "GarbageBefore\(child.name)"
+              : "GarbageBetween\(children[i - 1].name)And\(children[i].name)",
+            kind: "GarbageNodes",
+            isOptional: true,
+            collectionElementName: name
```

Comment:
`collectionElementName` should be the same as the parameter passed to `name` but right now it’s the node’s name.

Reply:
Good catch, thanks!